xcode/Subconscious/Shared/Components/Detail/UserProfileDetailMetaSheet.swift

```diff
@@ -0,0 +1,302 @@
+//
+//  UserProfileMetaSheet.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 29/3/2023.
+//
+
+import SwiftUI
+import ObservableStore
+
+struct MetaTableItemShareLinkView: View {
+    var label: String
+    var item: String
+    
+    var body: some View {
+        ShareLink(item: item) {
+            Label(
+                label,
+                systemImage: "square.and.arrow.up"
+            )
+            .labelStyle(RowLabelStyle())
+        }
+        .padding(
+            EdgeInsets(
+                top: 0,
+                leading: AppTheme.padding,
+                bottom: 0,
+                trailing: AppTheme.padding
+            )
+        )
+        .frame(height: AppTheme.unit * 11) // There has to be a better way...
```

Comment:
Can you say more about the why of this?

Reply:
I want to make use of `ShareLink` as the zero-stress way to present a share sheet, but to style a `ShareLink` so that appears consistent with buttons in the `MetaTable` required jumping through some hoops. However, I came up with a better solution (I think) if you take another look.

Reply:
Comment on this below.